reVX/least_cost_xmission/trans_cap_costs.py

```diff
@@ -926,6 +1033,11 @@ def compute_tie_line_costs(self, min_line_length=5.7,  # noqa: C901
         if save_paths:
             paths = []
 
+        if cost_layers is not None:
```

Comment:
Is `cost_layers` still optional with the new methodology?

Reply:
Nope, good catch. I will fix this